package.json

```diff
@@ -23,7 +23,7 @@
     "@angular/platform-browser": "2.4.10",
     "@angular/platform-browser-dynamic": "2.4.10",
     "@angular/router": "3.4.10",
-    "@ng-bootstrap/ng-bootstrap": "^1.0.0-alpha.21",
+    "@ng-bootstrap/ng-bootstrap": "1.0.0-alpha.21",
```

Comment:
this should have a comment as to why it is being pinned. Else, someone will walk into a trap!